src/app/applicationComponents/Header/index.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+import NewsIcon from '../../reusableComponents/svg/NewsIcon';
+
+import './Header.scss';
+
+const Header = () => (
+    <header className="site-header d-flex align-items-center flex-column">
+        <header className="d-flex align-items-center">
+            <span>
+                <NewsIcon height="150" width="150" />
+            </span>
+            News from all over the world
+        </header>
+        <nav className="site-header-navigation nav nav-pills d-flex justify-content-around">
```

Comment:
I think the app's navigation is a separate component, not a part of header

Reply:
Done. 
I have created NavigationPanel component.